python_binding/src/stream_writer.rs

```diff
@@ -137,12 +157,31 @@ impl StreamWriter {
         }
     }
 
+    ///
+    /// Flush all the inflight events into Pravega Stream.
+    /// This will ensure all the inflight events are completely persisted on the Pravega Stream.
+    ///
+    #[text_signature = "($self)"]
+    #[args("*")]
+    pub fn flush(&mut self) {
+        info!("Invoking flush() on writer {:?}", self.to_str());
+        for x in self.inflight.drain() {
+            let _res = self.factory.runtime().block_on(x);
```

Comment:
We are ignoring the oneshot result. What if the result is `Err`, I think we should bubble it up to the caller or throw an exception.

Reply:
yes, I have improved the code.